sdks/builds/android.mk

```diff
@@ -401,5 +401,5 @@ $(eval $(call AndroidCrossMXETemplate,cross-arm64-win,x86_64,aarch64-v8a,android
 $(eval $(call AndroidCrossMXETemplate,cross-x86-win,i686,i686,android-x86,llvm-llvmwin32,i686-none-linux-android))
 $(eval $(call AndroidCrossMXETemplate,cross-x86_64-win,x86_64,x86_64,android-x86_64,llvm-llvmwin64,x86_64-none-linux-android))
 
-$(eval $(call BclTemplate,android-bcl,monodroid monodroid_tools,monodroid))
+$(eval $(call BclTemplate,android-bcl,monodroid monodroid_tools net_4_x,monodroid))
```

Comment:
Add whatever tool you need to the `monodroid_tools` profile instead.

Reply:
The tool (jnimarshalmethod-gen.exe) comes from Java.Interop. Here we need the BCL assemblies, which work correctly on Windows.

Would it make more sense if I add BCL to monodroid_tools profile, compared to adding the net_4_x profile here?

Reply:
`monodroid_tools` is the profile used by XA to build things it will need to ship, and which is not part of the `monodroid` profile. Or put differently: `monodroid` is to run on the device, `monodroid_tools` is to run on the host. So feel free to add anything you require to it, it's only used by XA.